src/index.html

```diff
@@ -1,17 +1,180 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="form__fieldset">
+        <legend>Personal information</legend>
+        <div class="form__container">
+          <label for="surname">Surname:</label>
+          <input 
+          type="text" 
+          id="susname" 
+          name="surname" 
+          autocomplete="off" 
+          >
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) the 3rd point.
Fix attributes indents in all places
```suggestion
          <input 
	          type="text" 
	          id="susname" 
	          name="surname" 
	          autocomplete="off" 
          >
```